winml/lib/Api/LearningModel.cpp

```diff
@@ -165,11 +162,9 @@ LearningModel::GetOperatorRegistry() {
   auto operator_provider_native =
       operator_provider_.as<ILearningModelOperatorProviderNative>();
 
-  // Retrieve the "operator abi" registry.
-  winrt::com_ptr<IMLOperatorRegistry> operator_registry;
-  operator_provider_native->GetRegistry(operator_registry.put());
-
-  return operator_registry.get();
+  IMLOperatorRegistry* registry = nullptr;
+  adapter_->GetOperatorRegistry(operator_provider_native.get(), &registry);
```

Comment:
I think this needs to be wrapped with " WINML_THROW_IF_FAILED(" since adapter -> GetOperatorRegistry returns a HRESULT